src/allmydata/util/eliotutil.py

```diff
@@ -75,23 +60,6 @@
 )
 from twisted.application.service import Service
 
-def validateInstanceOf(t):
-    """
-    Return an Eliot validator that requires values to be instances of ``t``.
-    """
-    def validator(v):
-        if not isinstance(v, t):
-            raise ValidationError("{} not an instance of {}".format(v, t))
-    return validator
-
-def validateSetMembership(s):
-    """
-    Return an Eliot validator that requires values to be elements of ``s``.
-    """
-    def validator(v):
-        if v not in s:
-            raise ValidationError("{} not in {}".format(v, s))
-    return validator
```

Comment:
Hm.  These were used by magic-folder before it was split out into its own package.  I guess I shouldn't say we should keep them given that they're now unused.  We can always pull them back out of git history if we decide we want them ...

The larger story about logging in Tahoe-LAFS is that it leans heavily on Twisted and Foolscap logging and I'd prefer it largely switch to Eliot, though, so I suspect we will want these back.

Reply:
I can keep the Eliot stuff it you think it's likely to be used eventually.